templates/response_parser_template.php

```diff
@@ -169,8 +169,11 @@ private function _parseJsonObject(\$jsonEntry, \$fhirElementName) {
                         continue 2;
                 }
 
-                if (!isset(\$properties[\$k]))
-                {
+                // Ignore properties prefixed with an underscore
+                // Parsing the document fails with these properties
+                if (false !== strpos(\$k, '_')) {
```

Comment:
This is a fine short term solution, however lets change the statement to this:

` if (0 === strpos(\$k, '_')) ...` just to make sure we're only matching on prefixes.

Reply:
Done.